.github/workflows/lint.yml

```diff
@@ -61,3 +61,4 @@ jobs:
 
       - name: Compare default parameters
         run: ./scripts/compare_default_params.sh
+        continue-on-error: true
```

Comment:
Why is that needed? :)

Reply:
originally had the use_remote_postgres param in params.yml, but realized this leads to it needing to be copied to 3 param files (kurtosis / input_parser) which is a bit annoying.. this was old debugging line.. 

ultimately, spoke with john, and would be amazing to have singular config file for all use cases, if possible